docs/_sources/vc_features.rst.txt

```diff
@@ -0,0 +1,36 @@
+VC features
```

Comment:
Please move this file to `src/` rather than `docs/_sources` This will probably end up being written over. See https://github.com/conda-forge/conda-forge.github.io/blob/master/docs/README#L3